test/models/user_mailer_test.rb

```diff
@@ -178,11 +178,7 @@ def assert_support_present_in_text(link_text, email = @email)
     end
 
     should "mention that reset is disallowed because their account is suspended" do
-      assert_body_includes "You can't request a password reset on a suspended account."
-    end
-
-    should "include correct support links" do
-      assert_support_present_in_text "support form"
+      assert_body_includes "You cannot reset your GOV.UK Signon development account, for user@example.com while your account is suspended."
```

Comment:
Does this need an additional comma after the email address, i.e.:

> You cannot reset your GOV.UK Signon development account, for user@example.com, while your account is suspended.

?

Reply:
Yes, although on second thoughts it would read better as "You cannot reset the GOV.UK Signon password for user@example.com while the account is suspended." So remove all commas, add the word 'password' and change 'your' to 'the'. Sorry!

Reply:
Thanks both! I've updated the text as requested.